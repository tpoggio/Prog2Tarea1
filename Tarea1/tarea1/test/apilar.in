esVaciaPila
apilar 15 5.2
esVaciaPila
apilar 95 24.0
esVaciaPila
apilar 85 2.0
# puede haber repetidos
apilar 85 2.0
apilar 25 67.0
apilar 35 -12.0
apilar 75 2.5
apilar 45 2.1
apilar 55 -24.0
apilar 85 32.0
# esta llena
apilar 10 10.0
Fin
