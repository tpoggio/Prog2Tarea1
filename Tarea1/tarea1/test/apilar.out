1>La pila está vacia. 
2>Apilar. 
3>La pila no está vacia. 
4>Apilar. 
5>La pila no está vacia. 
6>Apilar. 
7>#  puede haber repetidos.
8>Apilar. 
9>Apilar. 
10>Apilar. 
11>Apilar. 
12>Apilar. 
13>Apilar. 
14>Apilar. 
15>#  esta llena.
16>Apilar. 
17>Fin.
